README.md

```diff
@@ -116,34 +116,113 @@ This allow to **configure** the layouts **on the go** while using it.
+9. Execute a command in each terminal instance so that the terminal windows are distinguishable from each other. For example, execute `ls, nano, man ls, htop` respectively.
+10. Use `Super+0` to open another workspace, and open two applications, for example, an email client and Libre Writer.
+11. Use `Super+End` to open another workspace, and  open an image editor.
+
+Practice
+
+1. Cycle through the tiling layouts using `Super+Spacebar`. Notice that the tiling-layout icon updates to represent the current layout.
+2. Experiment with keyboard shortcuts. Try the following:
+	-Move the focus within a workspace, to another workspace, and if applicable, to another monitor.
+	-Move a window within a workspace, to another workspace, and if applicable, to another monitor.
+	-Resize a window.
+	-Close an application.
+
+Take note of the following:
+
+1. The workspace icons are located on the left side of the screen and the window panes (tabs) are located at the top of the screen.
+2. When using a browser, the browser tabs are located below the window panes. Browser tabs and window panes can be similar in appearance.
+3. There isn't a keyboard/mouse equivalent of `Maximize` window. Use the tiling layout icon or `Super+spacebar` to return to a maximized window.
+4. Tiling layout preferences are unique to the each workspace (the preferences do not behave as workspace global settings).
+5. Note that a few Material-Shell keybindings duplicate some Gnome keybindings e.g. `Super+s, Super+PgDown`, `Super+Home`, `Super+End` etc..
+
 ## Hotkeys
-Some hotkeys might already be used by GNOME Shell - please check your keybindings first.
-#### Desktop navigation
-* `Super+W` Navigate to the upper workspace/category.
-* `Super+S` Navigate to the lower workspace/category.
-* `Super+A` Focus the window at the left of the current window.
-* `Super+D` Focus the window at the right of the current window.
-* `Super+1`, `Super+2` ... `Super+0` Navigate to specific workspace
-
-#### Window manipulation
-* `Super+Q` Kill the current window focused.
+Material-Shell is often used in conjunction with Gnome keyboard-shortcuts, Firefox shortcuts, etc. 
+Note that some hotkeys might already be used by other applications - please check your keybindings first.
+
+#### Choosing tiling layout
+* `Super+Space` Cycle through the tiling options in the current workspace.
+
+#### Open and close a window
+* `Super+x` Open the App Launcher within a new window.
+* `Super+q` Kill the current focused window.
+
+#### Moving the focus aka Navigation
+... window panes 
+* `Super+d` Focus on the next window pane, left to right.
+* `Super+a` Focus on the previous window pane, right to left.
+
+...workspaces 1 through 10
+* `Super+1`, `Super+2` ... `Super+0` Navigate to specific workspace 1 - 10
+* `Super+s` Focus on the next workspace 1 - 10 (down).
+* `Super+w` Focus on the previous workspace 10 - 1 (up).
```

Comment:
One of the most important concept of Material Shell is the spatial organisation and the navigation by direction (Navigating horizontally between windows and vertically between workspaces) that's why I would leave WASD shortcuts together.

Reply:
I understand what your getting now. I'm in my 3rd week of using M-S and the WASD keys now make sense. During the learning curve though I was constantly looking through the list for the correct combination to . . . move the focus, or move the window, or cycle through the layouts. After adding several additional kb combinations to the list it became more difficult to located specific combinations. The solution was to arrange by action so that the list becomes a Quick Reference. For example, I want to move Firefox to workspace 5. I don't have multiple monitors, therefore I can skip over that section entirely. I  quickly hone in on the subtitle "Move a Window". That was the thinking behind the groupings. There is an "Who is the intended audience?" question here. I suppose if WASD keys are a standard or historically significant, (coming from Win64, they're not to me) then the intended audience would likely pick up on it immediately. Then again a README is an introductory document and that audience would pick up on it immediately anyway. You know **Hotkeys Quick Reference** is a better title. 

Question. I had used Awesome for a few days and there was a floating-window hotkey help reference. What's the reason behind not creating hotkey help reference for M-S?